sympy/printing/printer.py

```diff
@@ -235,7 +235,7 @@ def doprint(self, expr):
         """Returns printer's representation for expr (as a string)"""
         return self._str(self._print(expr))
 
-    def _print(self, expr, *args, **kwargs):
+    def _print(self, expr, **kwargs):
```

Comment:
Can you explain this change, and the adding of `**kwargs` to the print methods? Does this break backwards compatibility for existing printer subclasses?

Reply:
Perhaps `*args` should be left intact, but when does a printer receive more than one positional argument to its `_print_*` methods? If there is such a case we should add a test case for it (demonstrating its use in e.g. a user-defined subclass).

The `**kwargs` are used for indentation when printing the ast nodes, see e.g.:
- https://github.com/sympy/sympy/pull/13100/files/717e9c2501e1ebb9c2a54c3992eb52e2a4dc357b#diff-0c241bae43013739d8f2e1ee4ae01c11R280
- https://github.com/sympy/sympy/pull/13100/files/717e9c2501e1ebb9c2a54c3992eb52e2a4dc357b#diff-0c241bae43013739d8f2e1ee4ae01c11R264